R/sfg.R

```diff
@@ -1,6 +1,5 @@
 # dim: what does the third dimension, if present, refer to? (XYZ or XYM)
 getClassDim = function(x, d, dim = "XYZ", type) {
-	stopifnot(d > 1 && d < 5)
```

Comment:
if not here, where is this sanity check done? What if this is not true?

Reply:
The same check is done on the last line of `getClassDim`.

Reply:
Then it doesn't make sense, indeed.